CIP-XXXX/README.md

```diff
@@ -0,0 +1,124 @@
+The most egregious of which is that an EBB occupies the same slot as its successor and has the same difficulty (ie BlockNo) as its predecessor.
+As an old, exotic, and useless class of block, EBBs have been initially overlooked in our subsequent design work many times and have ultimately wasted many hours of work as developers troubleshoot and ultimately determine how to accommodate EBBs via corner-cases etc.
+(TODO it would be possible to track down Issues, PRs, etc to partially evidence this.
+But a great deal of the trouble with EBBs has been time lost coping with them in discussions that were not recorded in a manner that demonstrates the EBBs' obstruction.
+Several former and current Consensus developers can attest to the trouble with EBBs, if that is deemed necessary.)
+
+It is therefore desirable to remove EBBs from the historical chain, so that the  complications they necessitate can be eliminated from existing _and new_ nodes' specifications and implementations, except for the small set of prev-hash overrides proposed by this CIP.
+
+However, doing so will disrupt some tooling that would otherwise expect the node to continue serving EBBs as part of the historical chain.
+Moreover, new nodes alternative to the venerable Haskell reference Cardano implementation are in development.
+Hence this CIP to open up the discussion of this adjustment of the historical chain to the community.
+
+## Specification
+
+The removal will proceed in stages.
+
+- In Stage One, nodes would be relaxed to allow upstream chains to omit EBBs, using a minor patch to the validation logic to allow these known finite exceptions to the prev-hash condition.
+  Regardless of whether it received EBBs when syncing the historical chain, this node would always store EBBs (even _ex nihilo_) and serve them to downstream nodes.
+
+- Once Stage One is sufficiently widespread, Stage Two would delete EBBs from immutable and volatile storage upon startup — aka _database migration_ — and therefore it would no longer serve EBBs to downstream peers.
+  This is when tooling such as db-sync, Mithril, etc would notice the change in behavior, which could be disruptive.
+
+- Finally, once Stage Two is sufficiently widespread, Stage Three would forbid upstream peers from sending EBBs.
+
+## Rationale: how does this CIP achieve its goals?
+
+The staging ensures interop between each successive stage, such that each Stage's node and its predecessor node will always be able to sync to/from one another.
+
+As of Stage Three, all nodes would relay a chain that contains no EBBs and would never store them.
+EBBs could now be entirely removed from the node specification and implementation, except for the prev-hash overrides described in this CIP.
+
+It would be possible to accelerate this plan to merely two stages, at the cost of the additional complexity necessary for the Stage One node to conditionally send EBBs depending on the mini protocol version negotiated with each downstream peer.
```

Comment:
@abailly I think this paragraph is the extent of what the current draft says relevant to your suggestion in this comment https://github.com/cardano-foundation/CIPs/pull/974/#issuecomment-2628056362

Reply:
Maybe I'm overlooking something? Let me know!

Reply:
FYI, I elaborated this section, adding the tables we were using in our other discussion.